address_to_census_tract.py

```diff
@@ -7,81 +7,167 @@
     ./address_to_census_tract.py
 
 Requirements:
-    geocoder
     shapely
 """
-import geocoder
 import json
 from shapely.geometry import shape, Point
-from sys import stderr
-
+from sys import stderr, argv
+import pandas as pd
+import os
+from smartystreets_python_sdk import StaticCredentials, exceptions, ClientBuilder
+from smartystreets_python_sdk.us_street import Lookup
+from smartystreets_python_sdk.us_extract import Lookup as ExtractLookup
 
 def main():
-    with open("data/test/testset.json", encoding = "UTF-8") as file:
+    if argv[1].endswith('.json'):
+        process_json(argv[1])  # hint: "data/test/testset.json"
+    elif argv[1].endswith('.csv'):
+        process_csv(argv[1])  # hint: "data/test/testset.csv"
+    else:
+        raise ValueError(f"Unknown file extension for file named «{argv[1]}». \
+Please choose from one of the following file extensions: \n \
+    * .csv \n \
+    * .json")
```

Comment:
FYI only, not necessary to change:

It can sometimes be nicer to write multi-line strings using Python's triple-quote syntax and [the `dedent` function from the textwrap module](https://docs.python.org/3/library/textwrap.html#textwrap.dedent). For example:

```python
        raise ValueError(dedent(f"""
            Unknown file extension for file named «{argv[1]}».
            Please choose from one of the following file extensions:
                * .csv
                * .json
        """))
```

Reply:
Good to know! I'll use that from now on.